Cataclysm/Druid.lua

```diff
@@ -549,9 +577,12 @@ spec:RegisterStateExpr("rip_refresh_time", function()
     return calc_rip_refresh_time
 end)
 
+--- Return the time at which Rip should be refreshed.
 spec:RegisterStateExpr("calc_rip_refresh_time", function()
+    local reaction_time = latency -- TODO: This appears to always be 0.1, a very low value, in order to clip the rip with TF the addon has a 0.2s window to perform this query before TF drops and then the user must react in the remaining time.
```

Comment:
Is `latency ` suppose to be the reaction time of the user?

Reply:
yeah, its used as a reaction time